### PR TITLE
WIP: left right mavigation drawer created

### PR DESCRIPTION
**Pull request by har2008preet**
### Summary of PR
Left and Right drawer added (discord like animation)

### Tests for unexpected behavior
- none_whatsoever

### Time spent on PR
~ 1 week

### Linked issues
Fix #

### CODE REVIEW

**Comment by har2008preet** [comment]
Currently having an issue with toggleDrawer with left drawer. It always toggles right drawer currently.

**Comment by Harjot1Singh** [comment]
> Currently having an issue with toggleDrawer with left drawer. It always toggles right drawer currently.

Did you need some help on this, or are currently working through it?

File: src/screens/RightDrawer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import Screens, { AppStackParams } from '../../lib/screens'
+import { gurbaniScreen } from '../Gurbani'
+import { searchScreen } from '../Search'
+
+import DrawerContainer from './DrawerContainer'
+
+const { Screen, Navigator } = createDrawerNavigator()
+const { Screen: StackScreen, Navigator: StackNavigator } = createStackNavigator<AppStackParams>()
+
+const screens = [ gurbaniScreen, searchScreen ]
```
Comment (Harjot1Singh) [inline src/screens/RightDrawer/index.tsx:14]: These screens technically aren't in the right drawer,  they're the content in the middle. The left and right drawers should push/overlay the centre content.

**Comment by har2008preet** [comment]
> > Currently having an issue with toggleDrawer with left drawer. It always toggles right drawer currently.
> 
> Did you need some help on this, or are currently working through it?

@Harjot1Singh Will work on this PR this coming Sunday. @Harjot1Singh (Work from home is over 😭 )

**Comment by Harjot1Singh** [comment]
Hey @har2008preet, since requirements have changed, I'm going to close this. If you do get a chance and want to get involved again, please reach out, we'd love to have you!

**Pull request closed without merging**